### ISSUE TITLE
Airtable Loader: Issues with `multipleLookupValues`

### ISSUE THREAD

**Issue by arabold** [ascorbic/astro-loaders#29]
Nice work with the `AirtableLoader`! I'm not using it yet but noticed a couple of small issues with the Zod schema creation when trying it earlier.

The `multipleLookupValues` needs to be handled like this to have the correct data types (i.e. if referencing a boolean value instead of strings in the lookup):

```ts
const airtableTypeToZodType = ({
  type,
  options,
}: AirtableField): ZodTypeAny => {
  // ...
  if (type === 'multipleLookupValues') {
    const resultType = options.result
      ? airtableTypeToZodType(options.result)
      : z.string();
    return z.array(resultType.optional().nullable());
  }

  const choices = options.choices?.map(({ name }) => name) as [
    string,
    ...string[],
  ];
}
```

Also note that `options.choices` is only set for enums, so adding a `?` will avoid crashes.

Again, great work so far! I didn't get around creating a fork and pull request yet, but I hope this helps and looking forward to potentially use the library in future.

**Comment by inbn** [comment]
I'm also getting an error in the `airtableTypeToZodType` function (when the `type` is `multipleRecordLinks`), which looks to be related: 
```
Cannot read properties of undefined (reading 'map')
```
on the line
```js
const choices = options.choices.map(({ name }) => name);
```

**Comment by ryanq** [comment]
I think this would be solved by #31. If you're feeling like diving into the plumbing, you can try that patch out in your project.

**Comment by inbn** [comment]
Happy to make a separate issue for this but I think url fields are still causing an error in the `airtableTypeToZodType` function because `options` is undefined. 

```
Exception has occurred: TypeError: Cannot read properties of undefined (reading 'choices')
```
I believe it's caused by the `if (options.choices) {` line.

Looking at the logic, it's probably because url fields don't have `options`. e.g.
```
{
    type: "url",
    id: "fldVovFBOo1KBKkhh",
    name: "URL",
},
```
